nucleus/payara-modules/hazelcast-bootstrap/src/test/java/fish/payara/nucleus/hazelcast/test/DNSDiscoveryTest.java

```diff
@@ -72,7 +72,7 @@ public void multipleRecordTest() {
     }
     
     @Test
-    public void sinlgeRecordTest() {
+    public void singleRecordTest() {
```

Comment:
Unrelated, just a typo fix

Reply:
Was this change made as a codebase wide refactor and not just a renaming of the method in the class? Assuming yes but always worth asking ;)

Reply:
No, but yes. I didn't check but luckily thats the only occurrence of the method